packages/sdk/src/evm/functions/core/cache.ts

```diff
@@ -0,0 +1,30 @@
+import { ValidContractInstance } from "../../core/types";
+import { ThirdwebStorage } from "@thirdweb-dev/storage";
+
+const CONTRACT_CACHE = new Map<string, ValidContractInstance>();
+
+const STORAGE_CACHE = new Map<"cache", ThirdwebStorage>();
+STORAGE_CACHE.set("cache", new ThirdwebStorage());
```

Comment:
this seems extremely odd, why would we immediately set something on the cache? did you want to just use a singleton?

Reply:
Yes this is very weird lmao not sure what I was doing here. Should just be a singleton lmao.

Reply:
fwiw I could see a usecase for multiple instances of storage